### ISSUE TITLE
Position after installation

### ISSUE THREAD

**Issue by georgringer** [TYPO3-Initiatives/dashboard#63]
If the extension is installed, the dashboard is the *first* module. should that be really so?

furthermore should it be really in pages and not in user section? does it need a page tree?

**Comment by NamelessCoder** [comment]
I agree here - the module should not be the first one. I also found myself clicking the dashboard module icon when I wanted to view the page module, since in every TYPO3 site ever, the page module has come first.

I would put it in "user" and leave it up to the site administrator to make it the default module for a user(group).

**Comment by georgringer** [comment]
@NamelessCoder once this is in core we could have set it as the default module for new users, what you think?

**Comment by haassie** [comment]
It will be the startup screen by default if you ask me. I would love to move it somewhere else, but the user section is really to "hidden". A dashboard should be accessible easily and you should not need multiple clicks to reach it. Do you have any suggestion?

**Comment by NamelessCoder** [comment]
I like that TYPO3 currently has a per-user "starting module" setting which loads that module as the first one. I would not 1) remove this support or 2) change the default setting.

Here's what we could do:

1. Instead of users' UC defaulting to empty selection, we could have it default to a global configuration parameter.
2. But instead of giving this global configuration parameter a default value (which would change current "use first" behavior for existing sites) we leave the default empty.
3. Then in the install tool, present a selection for "starting module".
4. And equally make this possible to change through BE "settings" module.

Relevant core code: `\TYPO3\CMS\Backend\Controller\BackendController::setStartupModule`.

In short: make it opt-in, allow a site-administrator to change the default behavior from "use first" to "use specific module".

**Comment by NamelessCoder** [comment]
Don't get me wrong, I think the dashboard idea is great but don't loose sight that the "page" module is still the absolutely central place of operation for everyday users. In essence it's the "CMS" part of TYPO3.

Another perhaps viable option: if EXT:dashboard is installed, let EXT:backend render the overview for page UID=0 which doesn't have a lot of usefulness currently.

Maybe you'd then even want to drop the backend module entirely and make the behavior that you get a dashboard on pid zero in page module?

**Comment by NamelessCoder** [comment]
![slice1](https://user-images.githubusercontent.com/2228052/67226987-9236ff80-f436-11e9-947f-72ad7f11f725.png)

Just to show what I'm thinking.

**Comment by haassie** [comment]
We were thinking about placing it in the top bar as an icon with text. Would that be an option for you? The issue is that having it on uid=0 is not very intuitive for users.

**Comment by NamelessCoder** [comment]
I like the idea of making this a top bar icon alongside cache, user etc.

**Comment by haassie** [comment]
![Screenshot 2019-10-23 at 17 02 38](https://user-images.githubusercontent.com/11075733/67406874-f54fa000-f5b6-11e9-8fa3-ddef8cc88ad5.png)

Would something like this be ok @irenesacchi ? See the icon + text in the top bar.

**Comment by irenesacchi** [comment]
I think the idea with the icon and the text on the top bar is a good one. Still I think it is important for a lot of users to get the dashboard as first screen. In case of the developers that do not need the dashboard they can still organize their backend in a different way. But yes, if we hide the dashboard in the three it loses its purpose. The top btn is still a very good option to “go back” to the dashboard screen without searching for it.
